phantom-finagle/src/main/scala/com/websudos/phantom/finagle/package.scala

```diff
@@ -0,0 +1,450 @@
+        set => ListResult(directMapper(set.all), set)
+      }) (state => query.execute(_.setPagingState(state)) map {
+        set => ListResult(directMapper(set.all), set)
+      })
+    }
+}
+
+  implicit class BatchQueryAugmenter[ST <: ConsistencyBound](val batch: BatchQuery[ST]) extends AnyVal {
+    def execute()(implicit session: Session, keySpace: KeySpace): Future[ResultSet] = {
+      twitterQueryStringExecuteToFuture(batch.makeBatch())
+    }
+  }
+
+  implicit class ExecutableStatementListAugmenter(val list: ExecutableStatementList) extends AnyVal {
+    def execute()(implicit session: Session, keySpace: KeySpace): Future[Seq[ResultSet]] = {
+      Future.collect(list.queries.map(item => {
+        twitterQueryStringExecuteToFuture(new SimpleStatement(item.terminate().queryString))
+      }))
+    }
+  }
+
+
+
+  implicit class CreateQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Status <: ConsistencyBound
+  ](val query: CreateQuery[Table, Record, Status]) extends AnyVal {
+
+    def execute()(implicit session: Session, keySpace: KeySpace): Future[ResultSet] = {
+      if (query.table.secondaryKeys.isEmpty) {
+        twitterQueryStringExecuteToFuture(new SimpleStatement(query.qb.terminate().queryString))
+      } else {
+        query.execute() flatMap {
+          res => {
+            query.indexList(keySpace.name).execute() map {
+              _ => {
+                Manager.logger.debug(s"Creating secondary indexes on ${QueryBuilder.keyspace(keySpace.name, query.table.tableName).queryString}")
+                res
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  implicit class ExecutableCreateStatementsListAugmenter(val list: ExecutableCreateStatementsList) extends AnyVal {
+    def execute()(implicit session: Session, keySpace: KeySpace): Future[Seq[ResultSet]] = {
+      Future.collect(list.tables.toSeq.map(_.create.ifNotExists().execute()))
+    }
+  }
+
+  implicit class InsertQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Status <: ConsistencyBound,
+    PS <: HList
+  ](val query: InsertQuery[Table, Record, Status, PS]) extends AnyVal {
+    def ttl(duration: com.twitter.util.Duration): InsertQuery[Table, Record, Status, PS] = {
+      query.ttl(duration.inSeconds)
+    }
+  }
+  implicit class SelectQueryAugmenter[
+    Table <: CassandraTable[Table, _],
+    Record,
+    Limit <: LimitBound,
+    Order <: OrderBound,
+    Status <: ConsistencyBound,
+    Chain <: WhereBound,
+    PS <: HList
+  ](val select: SelectQuery[Table, Record, Limit, Order, Status, Chain, PS]) extends AnyVal {
+    /**
+      * Returns the first row from the select ignoring everything else
+      * This will always use a LIMIT 1 in the Cassandra query.
+      * @param session The Cassandra session in use.
+      * @return
+      */
+    @implicitNotFound("You have already defined limit on this Query. You cannot specify multiple limits on the same builder.")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/flavian/phantom/file/2525508486/issues/source?bid=2847375&fileBranchId=3265057#l336)